packages/gestalt/src/Controller.js

```diff
@@ -127,10 +131,11 @@ export default class Controller extends Component<Props, State> {
   }
 
   handleKeyDown: (event: {| keyCode: number |}) => void = (event) => {
-    const { onDismiss } = this.props;
+    const { handleKeyDown, onDismiss } = this.props;
     if (event.keyCode === ESCAPE) {
       onDismiss();
     }
+    if (handleKeyDown) handleKeyDown(event);
```

Comment:
I dont' think we support this format in Pinboard, any thought?

![image](https://user-images.githubusercontent.com/10593890/102563067-bc656b00-4095-11eb-8c6c-d4e5cf4283f4.png)

Reply:
We don't, and I don't particularly like it, but I didn't comment on it as that's not anything specific to this PR. That's something we should change via the lint rule and autofix / codemod.